.github/workflows/build.yml

```diff
@@ -113,6 +113,55 @@ jobs:
         git -C taps/ push
 
 
+  build_unix_arm:
+      needs: prepare
+      runs-on: ubuntu-18.04  # Standalone executable should be built on minimum supported OS
+      strategy:
+        matrix:
+          architecture:
+          - armv7
+          - aarch64
+
+      steps:
+      - uses: actions/checkout@v3
+        with:
+          path: ./repo
+
+      - name: Virtualized Install, Prepare & Build
+        uses: uraimo/run-on-arch-action@v2
+        with:
+          arch: ${{ matrix.architecture }}
+          distro: ubuntu18.04
+
+          setup: |
+            mkdir -p "${PWD}/artifacts"
+
+          # Setup shared volume folders between the host container and the child container
+          dockerRunArgs: |
+            --volume "${PWD}/repo:/repo"
+            --volume "${PWD}/artifacts:/artifacts"
+            
+          # Additional shell commands to be executed in the docker container as part of docker build. Will be included in the cached docker image if githubToken is set 
+          install: |
+            apt update
+            apt -y install zlib1g-dev python3.8 python3.8-dev python3.8-distutils python3-pip
+
+          # The installation, preparation and build steps have been merged together since there's only a single run step in the action
+          run: |
+            cd repo
+            python3.8 -m pip install Pyinstaller -r requirements.txt
+            python3.8 devscripts/update-version.py ${{ needs.prepare.outputs.version_suffix }}
+            python3.8 devscripts/make_lazy_extractors.py
+            python3.8 pyinst.py
+            cp dist/yt-dlp_linux* ../artifacts/yt-dlp_${{ (matrix.architecture == 'armv7' && 'armv7l') || matrix.architecture }}_linux
```

Comment:
just calling it arm7 is good enough imho. But I wont insist

Reply:
It has to be called armv7l to correspond with the value returned by Python's platform.machine() return value